roles/openshift_metering/files/metering-helm-operator-deployment.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: metering-helm-operator
+  labels:
+    app: metering-helm-operator
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: metering-helm-operator
+  template:
+    metadata:
+      labels:
+        app: metering-helm-operator
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      containers:
+      - name: metering-helm-operator
+        image: "quay.io/coreos/chargeback-helm-operator:0.6.1-latest"
+        args: ["run-operator.sh"]
+        imagePullPolicy: Always
+        env:
+        - name: HELM_RELEASE_CRD_NAME
+          value: Metering
+        - name: HELM_RELEASE_CRD_API_GROUP
+          value: chargeback.coreos.com
+        - name: HELM_CHART_PATH
+          value: /openshift-metering-0.1.0.tgz
+        - name: MY_POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: MY_POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: HELM_HOST
+          value: "127.0.0.1:44134"
+        - name: HELM_WAIT
+          value: "false"
+        - name: HELM_RECONCILE_INTERVAL_SECONDS
+          value: "30"
+        - name: RELEASE_HISTORY_LIMIT
+          value: "3"
+        resources:
+          requests:
+            memory: "25Mi"
+            cpu: "50m"
+          limits:
```

Comment:
Request defaults make sense, but do we need limits? With Prometheus, for example, we want dynamic growth to be possible for maximum portability across our various environments at different scales.

Reply:
I try to set both since quotas would do it if we don't, and it puts us into the guaranteed QOS. It shouldn't really need more I don't think. This is a deployment simple operator just doing templating using helm basically.